src/vm/amd64/JitHelpers_InlineGetThread.asm

```diff
@@ -1128,7 +1135,8 @@ ifdef TRACK_SYNC
         add     rsp, MON_ENTER_STACK_SIZE_INLINEGETTHREAD
 endif
 endif
-        mov     rdx, [rsp + 10h]
```

Comment:
It is quite possible that the `MON_DEBUG+TRACK_SYNC` path is broken. I do not think anybody used it for years.